### PR TITLE
feat: adding log export routes

### PR DESCRIPTION
**Pull request by csgulati09**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -9105,6 +9107,305 @@ paths:
                 schema:
                   $ref: "#/components/schemas/WorkspaceWithUsers"
 
+    /v2/logs/exports/{exportId}:
```
Comment (VisargD) [inline openapi.yaml:9110]: This spec is used to document user facing APIs. So it should be /v1 instead of /v2. We will internally route it to /v2

Comment (csgulati09) [reply openapi.yaml:9110]: Oh okay. 
So I am just remove v2, as done in other routes

File: openapi.yaml [hunk 2]
```diff
@@ -9105,6 +9107,305 @@ paths:
...
+            application/json:
+              schema:
+                type: object
+                properties:
+                  workspace_id:
+                    type: string
+                  filters:
+                    type: object
+                    properties:
+                      organisation_id:
```
Comment (sk-portkey) [inline openapi.yaml:9264]: move this to a reusable common Schema

**Review by sk-portkey — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 2 above)
Comment (csgulati09) [reply openapi.yaml]: Done

File: openapi.yaml [hunk 3]
```diff
@@ -15959,6 +16114,181 @@ components:
...
+              type: string
+            metadata:
+              type: object
+              additionalProperties:
+                type: string
+            prompt_id:
+              type: array
+              items:
+                type: string
+            ai_org_model:
```
Comment (sk-portkey) [inline openapi.yaml:16267]: can you recheck the type here. it should be array string

**Review by sk-portkey — CHANGES_REQUESTED** [review]

**Review by sk-portkey — APPROVED** [review]

**Pull request merged**